creator/studies/mutations.py

```diff
@@ -736,6 +736,64 @@ def mutate(self, info, study, data):
         return UpdatePhenotypeStatusMutation(study=study)
 
 
+class TransferStudyMutation(graphene.Mutation):
+    """ Mutation to transfer study to another organization"""
+
+    class Arguments:
+        study = graphene.ID(
+            required=True,
+            description="The ID of the study to remove the collaborator from",
+        )
+        organization = graphene.ID(
+            required=True,
+            description="The organization to transfer the study to."
+        )
+
+    study = graphene.Field(StudyNode)
+
+    def mutate(self, info, study, organization):
+        """
+        Update the organization of a study
+        """
+        user = info.context.user
+        if not user.has_perm("organizations.change_organization"):
```

Comment:
We can be a little more safe and also check that the user belongs to both the source and destination organizations. 
https://github.com/kids-first/kf-api-study-creator/pull/682/files#diff-efc346caa0fbfd8a6e993133f86fd9ab49ae702faff904ad698a9ae51b73361dR74